scripts/exgfs_atmos_postsnd.sh

```diff
@@ -35,7 +35,7 @@ export ENDHOUR=${ENDHOUR:-180}
 export INCREMENT=12
 export MAKEBUFR=NO
 export F00FLAG=YES
-export fformat=${OUTPUT_FILE:-netcdf}
+export fformat=netcdf
```

Comment:
Is this used downstream somewhere we can't edit?

Reply:
It is used in required parameter for construct namelist.

Reply:
Well, the heredoc for the namelist in `ush/gfs_bufr.sh` could be updated. But this is close enough for now.